website/blog/2023-3-23-malware.md

```diff
@@ -0,0 +1,108 @@
+---
+slug: malware-detection
+title: "Microsoft 365, Malware 👾, and your responsibilities"
+description: "By watching my
+four short videos, you’ll be set up and on your way to backing up all your Microsoft 365
+data in less than 20 minutes. That’s shorter than most meetings!"
+authors: nica
+tags: [corso, microsoft 365, backups, security, malware]
+date: 2023-3-23
+image: ./images/invaders.png
+---
+
+![a clone of the game 'space invaders' Cover image By Lee Robinson - https://github.com/leerob/space-invaders, MIT, https://commons.wikimedia.org/w/index.php?curid=127314893](./images/invaders.png)
+
+When we evaluate malware threats, we often think mainly of protecting our users.
+The biggest concern is always going to be lost availability and leaked data if
+malware affects our system. But like any threat with an infection model, part of
+the story is about your responsibilities as an operations engineer to keep
+others safe.
+<!-- truncate -->
+
+## Microsoft-hosted Malware
+
+Research earlier in 2023 showed that Microsoft OneDrive was
+[host to about 30% of all malware](https://www.cybertalk.org/2023/01/27/do-you-use-onedrive-or-google-drive-watch-out-for-this-malware/).
+OneDrive is a popular platform for hosting malware because the malicious actor
+can get a legitimate looking URL that will increase the chance of their payload
+being downloaded or executed.
+
+This malware hosting is usually done on accounts created by malicious actors,
+but it's even more effective if an compromised account within a legitimate
+organization can be used
+
+The responsibility for addressing this issue lies more with administrators than
+with Microsoft. Data on OneDrive is after all customer data so at the end of the
+day MSFT it's not likely that MSFT will automatically start taking down files.  
```

Comment:
```suggestion
with Microsoft. Data on OneDrive is customer data ant it will be very intrusive and disruptive 
for Microsoft to automatically start taking down files.  
```